packages-web/calendar/tests/TestProjects/Mendix8/theme/styles/native/core/helpers/maps.js

```diff
@@ -0,0 +1,56 @@
+import { border, brand } from "../variables";
+
```

Comment:
Remove native them, not needed, is pollutes the commit and repo

Reply:
Done